index.bs

```diff
@@ -108,8 +121,8 @@ The ProximitySensor Interface {#proximity-sensor-interface}
   };
 </pre>
 
-To <dfn>Construct a ProximitySensor Object</dfn> the user agent must invoke the
-<a>construct a Sensor object</a> abstract operation.
+To <dfn>construct a {{ProximitySensor}} Object</dfn> the user agent must invoke the
```

Comment:
We have dropped this `<dfn>` from other concrete specs since it is not referenced from anywhere, so could say:

`To construct a {{ProximitySensor}} object the user agent must invoke the` (with lowercase object)

Reply:
Thanks! Done.